qiskit/providers/honeywell/honeywellprovider.py

```diff
@@ -0,0 +1,218 @@
+#   Copyright 2019-2020 Honeywell, Intl. (www.honeywell.com)
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Provider for a Honeywell backends."""
+
+import logging
+from collections import OrderedDict
+
+from qiskit.providers import BaseProvider
+from qiskit.providers.models import BackendConfiguration
+
+from .api import HoneywellClient
+from .credentials import discover_credentials, write_creds_to_qiskitrc, remove_creds_from_qiskitrc
```

Comment:
Where is this module defined? I don't see a `credentials` module in `qiskit/providers/honeywell/`?